apps/meteor/app/statistics/server/lib/statistics.ts

```diff
@@ -157,6 +158,12 @@ export const statistics = {
 				}),
 		);
 
+		// Number of livechat rooms with priorities
+		statistics.totalLivechatRoomsWithPriority = await LivechatRooms.countLivechatRoomsByPriority();
+
+		// Number of livechat rooms with department
+		statistics.totalLivechatRoomsWithDepartment = await LivechatRooms.countLivechatRoomsWithDepartment();
```

Comment:
Maybe you can add these 2 like the one in L168, by pushing the promise to the array of promises instead of doing them 1 by 1

Reply:
Good idea, I'll do that, thanks :D